docs/Presentations/OSR-Roundtable.md

```diff
@@ -0,0 +1,102 @@
+---
+title: OSR Roundtable at OSFF Oct 2024
+sidebar_label: OSFF-24
+sidebar_position: 10
```

Comment:
Should we not position this the highest?

Reply:
I've put them in the sidebar in the chronological order they happened.

Reply:
Ah. Hoped for american chronological order :D (eg newest the top)